exp-player/addon/components/exp-overview/component.js

```diff
@@ -107,13 +107,16 @@ export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-overview',
     layout: layout,
     questions: questions,
-    responses: Ember.computed('questions', function() {
-        var questions = this.get('questions');
-        var responses = {};
-        for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
-        }
-        return responses;
+    showOptional: Ember.computed('questions.7.value', function() {
+       return this.questions[7].value === 'global.yesLabel';
+    }),
+    responses: Ember.computed(function() {
```

Comment:
Incidentally, I'm using `questions.n.value` in an ember.alias, and it appears to be tracking the change so far. 
https://github.com/CenterForOpenScience/exp-addons/pull/131/files#diff-20e4394de68cd99661ed9d994056e5a6R112

However, I agree that the lack of documentation around it makes me nervous, and I'm watching for edge cases as I build out around this.